routers/api/v1/admin/user.go

```diff
@@ -505,15 +505,14 @@ func RenameUser(ctx *context.APIContext) {
 	}
 
 	newName := web.GetForm(ctx).(*api.RenameUserOption).NewName
-
-	if strings.EqualFold(newName, ctx.ContextUser.Name) {
+	if newName == ctx.ContextUser.Name {
 		// Noop as username is not changed
 		ctx.Status(http.StatusNoContent)
 		return
 	}
 
 	// Check if user name has been changed
-	if err := user_service.RenameUser(ctx, ctx.ContextUser, newName); err != nil {
+	if err := user_service.RenameUser(ctx, ctx.ContextUser, newName, strings.EqualFold(newName, ctx.ContextUser.Name)); err != nil {
```

Comment:
Sometimes `EqualFold`, sometimes `strings.ToLower`, could there be a unique function?